src/twisted/application/twist/_options.py

```diff
@@ -59,7 +60,15 @@ def opt_reactor(self, name):
         The name of the reactor to use.
         (options: {options})
         """
-        self["reactorName"] = name
+        # Actually actually actually install the reactor right at this very
+        # moment, before any other code (for example, a sub-command plugin)
+        # runs and accidentally imports and installs the default reactor.
+        try:
+            self["reactor"] = self.installReactor(name)
```

Comment:
This is not the right method to install the reactor in, because this command line will not work:

```
twist --reactor X --reactor Y
```

This should install Y, but here X will get installed first, then probably raise an error when it tries to install Y.